webpack.config.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2007-2015, GoodData(R) Corporation. All rights reserved.
```

Comment:
I've taken it from some webpack grunt integration example and I kinda like it - the common config for webpack is separated and the differences for each grunt subtask are overridden in Gruntfile.js